hooks/pre-commit.sh

```diff
@@ -50,7 +50,7 @@ fi
 # Format all staged files, then exit with an error code if any have uncommitted
 # changes.
 echo 'Formatting staged Python files . . .'
-yapf -i -r ${PYTHON_FILES[@]}
+npm run yapf:fix
```

Comment:
I think it would be better to leave it as `yapf -i -r ${PYTHON_FILE[@]}`
Doesn't make sense to me, why this should be dependent on npm or anything else.